verification/example_config.toml

```diff
@@ -6,3 +6,6 @@ addr = "0.0.0.0:8043"
 api_url = "https://sourcify.dev/server/"
 verification_attempts = 3
 request_timeout = 10
+
+[compiler]
+compilers_list_url = "https://binaries.soliditylang.org/linux-amd64/"
```

Comment:
What if we rename `[verifier]` section into `[solidity]` (if vyper to be added, it would be another config anyway) and move the parameter into `[solidity]`?